app/jobs/process_retrieval_job.rb

```diff
@@ -59,7 +49,7 @@ def perform(retrieval_id, base_url, cmr_env)
               search_params: params
             )
 
-            SubmitLegacyServicesJob.set(queue: queue_name).perform_later(
+            SubmitLegacyServicesJob.set(queue: 'legacy_services').perform_later(
```

Comment:
You can define `SubmitLegacyServicesJob::queue_as` and then avoid having to set the queue anywhere else.